pkg/filetree/depth_first_path_walker.go

```diff
@@ -59,11 +59,12 @@ func (w *DepthFirstPathWalker) Walk(from file.Path) (file.Path, *filenode.FileNo
 	w.pathStack.Push(from)
 
 	var currentPath file.Path
-	var currentNode *filenode.FileNode
+	var currentNode *nodeAccess
 	var err error
 
 	for w.pathStack.Size() > 0 {
 		currentPath = w.pathStack.Pop()
+		// TODO: should we make these link resolutions configurable so you can observe the links on walk as well?
```

Comment:
Is this TODO still a candidate for this PR or is it a future consideration? 

What do you mean by configurable? Is it a setting where instead `true` are replaced by some user provided value?